README.md

```diff
@@ -31,6 +31,23 @@ Or install it yourself as:
 
     $ gem install dep-selector-libgecode
 
+## Installation on Windows
+
+* Don't set USE_SYSTEM_GECODE to anything. The build only checks for the
+  presence of this variable, not the value. If you really want to install
```

Comment:
This second sentence that begins with "If you really want to install..." can be removed.